### ISSUE TITLE
Draw focus and include type in tab order for account edit

### ISSUE THREAD

**Issue by misterhaan** [misterhaan/abe#70]
Adding an account with a keyboard is difficult because most fields don’t have a visual focus indicator, and the account type selector doesn’t have a tab stop.
